js/apps/admin-ui/src/realm-settings/GeneralTab.tsx

```diff
@@ -251,79 +247,40 @@ export const RealmSettingsGeneralTab = ({
             )}
           />
         </FormGroup>
-        {isFeatureEnabled(Feature.DeclarativeUserProfile) && (
-          <FormGroup
-            hasNoPaddingTop
-            label={t("userProfileEnabled")}
-            labelIcon={
-              <HelpItem
-                helpText={t("userProfileEnabledHelp")}
```

Comment:
This label is no longer used either, so it can be removed from the properties as well.